tutorials/roofit/rf311_rangeplot.py

```diff
@@ -22,17 +22,17 @@
 gx = ROOT.RooGaussian("gx", "gx", x, ROOT.RooFit.RooConst(0), ROOT.RooFit.RooConst(1))
 gy = ROOT.RooGaussian("gy", "gy", y, ROOT.RooFit.RooConst(0), ROOT.RooFit.RooConst(1))
 gz = ROOT.RooGaussian("gz", "gz", z, ROOT.RooFit.RooConst(0), ROOT.RooFit.RooConst(1))
-sig = ROOT.RooProdPdf("sig", "sig", ROOT.RooArgList(gx, gy, gz))
+sig = ROOT.RooProdPdf("sig", "sig", [gx, gy, gz])
 
 # Create background pdf poly(x)*poly(y)*poly(z)
-px = ROOT.RooPolynomial("px", "px", x, ROOT.RooArgList(ROOT.RooFit.RooConst(-0.1), ROOT.RooFit.RooConst(0.004)))
-py = ROOT.RooPolynomial("py", "py", y, ROOT.RooArgList(ROOT.RooFit.RooConst(0.1), ROOT.RooFit.RooConst(-0.004)))
+px = ROOT.RooPolynomial("px", "px", x, [ROOT.RooFit.RooConst(-0.1), ROOT.RooFit.RooConst(0.004)])
+py = ROOT.RooPolynomial("py", "py", y, [ROOT.RooFit.RooConst(0.1), ROOT.RooFit.RooConst(-0.004)])
```

Comment:
```suggestion
px = ROOT.RooPolynomial("px", "px", x, [-0.1, 0.004])
py = ROOT.RooPolynomial("py", "py", y, [0.1, -0.004])
```

With https://github.com/root-project/root/pull/8607, it became possible to pass constants directly to a RooArgList without wrapping them with RooConst(). You should make use of this.